system/jlib/jptree.cpp

```diff
@@ -7860,16 +7860,15 @@ static void applyCommandLineOption(IPropertyTree * config, const char * option)
     }
 }
 
-static void displayConfig(IPropertyTree * config, const char * componentTag)
+jlib_decl StringBuffer & displayConfig(StringBuffer &jsonText, IPropertyTree * config, const char * componentTag)
```

Comment:
picky: Should really rename the function now it isn't displaying it.

Reply:
renamed